x-team/xp-ui
# Remove VideoPlayer from xp-registration dependencies

**Release Type:** *Non breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3362

## Description

Remove VideoPlayer from xp-registration dependencies

## Checklist


## Related PRs

branch | PR
------ | ------
`XP-3362-strip-out-record-functionality` | [XP](https://github.com/x-team/xp/pull/2006)
`XP-3362-strip-out-record-functionality` | [XP API](https://github.com/x-team/xp-api/pull/714)
`XP-3362-strip-out-record-functionality` | [XP Registration](https://github.com/x-team/xp-registration/pull/76)

## Steps to Test or Reproduce

1. Copy `XP-UI` bundle to `XP-Reg/Site` (using related branches)
1. Run `XP-Reg/Site`
1. Observe that nothing breaks
